src/STW.StubApi.Presentation/Services/Bcp/BcpController.cs

```diff
@@ -0,0 +1,43 @@
+namespace STW.StubApi.Presentation.Services.Bcp;
+
+using Microsoft.AspNetCore.Mvc;
+using Models;
+
+[ApiController]
+[Route("bcps")]
+public class BcpController
+{
+    private static readonly List<BcpSearchResultDto.BcpDto> Bcps =
+    [
+        new BcpSearchResultDto.BcpDto
+        {
+            Id = 1,
```

Comment:
These might be better placed in another file so we don't clutter the controller.